src/containers/pages/MDAPoint/ChildReports/index.tsx

```diff
@@ -0,0 +1,221 @@
+} from '../../../../store/ducks/generic/MDAChildReport';
+import { getMDAPointPlanById } from '../../../../store/ducks/generic/MDAPointPlans';
+import { ChildSupersetDataTable, ChildSupersetDataTableProps } from './ChildSupersetDataTable';
+/** register the MDA point child report definitions reducer */
+reducerRegistry.register(MDAPointChildReportReducerName, MDAPointChildReportReducer);
+
+const slices = SUPERSET_MDA_POINT_REPORTING_JURISDICTIONS_DATA_SLICES.split(',');
+
+interface ChildReportsProps extends ChildSupersetDataTableProps {
+  pageTitle: string | null;
+  pageUrl: string;
+  prevPage: Page[] | null;
+}
+
+const tableHeaders = [
+  CHILDS_NAME,
+  NATIONAL_ID,
+  ENROLLED_IN_SCHOOL,
+  MMA_DRUGS_ADMINISTRED,
+  SACS_REFUSED,
+  SACS_SICK,
+  ADR_REPORTED,
+  PZQ_DISTRIBUTED,
+  ALB_TABLETS_DISTRIBUTED,
+];
+
+/*
+ * Renders a table list of child report
+ * @param {ChildReportsProps} props
+ */
+const ChildReportList = (props: ChildReportsProps) => {
+  const {
+    pageTitle,
+    pageUrl,
+    supersetSliceId,
+    fetchItems,
+    service,
+    data,
+    headerItems,
+    tableClass,
+    prevPage,
+  } = props;
+
+  const homePage = [
+    {
+      label: HOME,
+      url: HOME_URL,
+    },
+    {
+      label: MDA_POINT_PLANS,
+      url: REPORT_MDA_POINT_PLAN_URL,
+    },
+  ];
+
+  const pages = prevPage ? [...homePage, ...prevPage] : [...homePage];
+  const breadcrumbProps = {
+    currentPage: {
+      label: pageTitle || MDA_POINT_CHILD_REPORT_TITLE,
+      url: pageUrl,
+    },
+    pages,
+  };
+
+  const listViewProps: ChildSupersetDataTableProps = {
+    data,
+    fetchItems,
+    headerItems,
+    service,
+    supersetSliceId,
+    tableClass,
+  };
+  const fullTitle = `${MDA_POINT_CHILD_REPORT_TITLE}: ${pageTitle}`;
+
+  return (
+    <div>
+      <Helmet>
+        <title>{fullTitle}</title>
+      </Helmet>
+      <HeaderBreadcrumb {...breadcrumbProps} />
+      <Row>
+        <Col md={8}>
+          <h3 className="mt-3 mb-3 page-title">{fullTitle}</h3>
+        </Col>
+      </Row>
+      <Row>
+        <Col>
+          <ChildSupersetDataTable {...listViewProps} />
+        </Col>
+      </Row>
+    </div>
+  );
+};
+
+/** Declare default props for MDAPointPlansList */
+const defaultProps: ChildReportsProps = {
+  data: [],
+  fetchItems: FetchMDAPointChildReportAction,
+  headerItems: tableHeaders,
+  pageTitle: null,
+  pageUrl: MDA_POINT_CHILD_REPORT_URL,
+  prevPage: [],
+  service: supersetFetch,
+  supersetSliceId: SUPERSET_MDA_POINT_CHILD_REPORT_DATA_SLICE,
+  tableClass: 'table table-striped table-bordered plans-list',
+};
+
+ChildReportList.defaultProps = defaultProps;
+
+export { ChildReportList };
+
+interface DispatchedStateProps {
+  data: React.ReactNode[][];
+  pageUrl: string;
+  pageTitle: string | null;
+  prevPage: Page[];
+}
+
+/** map state to props */
+const mapStateToProps = (
+  state: Partial<Store>,
+  ownProps: RouteComponentProps<RouteParams>
```

Comment:
lets use the full type even if we are only interested in the routeComponent props